### ISSUE TITLE
Swackets Package v0.12.0 does not activate when starting Atom 1.12.0

### ISSUE THREAD

**Issue by amandeepmittal** [jrattray13/swackets#27]
Gives an error:

```
Failed to activate the swackets package
Invalid regular expression: /(?<!\\)(?:[$%#].?)?[\{]/: 
Invalid group
```
Show Stack Race:
```
SyntaxError: Invalid regular expression: /(?<!\\)(?:[$%#].?)?[\{]/: Invalid group
    at RegExp (native)
    at SwacketsView.module.exports.SwacketsView.config (/Users/amandeepmittal/.atom/packages/swackets/lib/swackets-view.coffee:64:26)
    at new SwacketsView (/Users/amandeepmittal/.atom/packages/swackets/lib/swackets-view.coffee:13:19)
    at Object.activate (/Users/amandeepmittal/.atom/packages/swackets/lib/swackets.coffee:32:21)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:184:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:157:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:150:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:548:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:529:29
    at Config.module.exports.Config.transactAsync (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:524:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:506:46)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:783:28
```

Atom Version: `1.12.0`
macOS :`10.12.1`
Thrown From: swackets package, `v0.12.0`

**Comment by petruciucur** [comment]
Error on Atom 1.12.0

SyntaxError: Invalid regular expression: /(?<!\\)(?:[$%#].?)?[\{]/: Invalid group
    at RegExp (native)
    at SwacketsView.module.exports.SwacketsView.config (file:///C:/Users/Pety/.atom/packages/swackets/lib/swackets-view.coffee:64:26)
    at new SwacketsView (file:///C:/Users/Pety/.atom/packages/swackets/lib/swackets-view.coffee:13:19)
    at Object.activate (file:///C:/Users/Pety/.atom/packages/swackets/lib/swackets.coffee:32:21)
    at Package.module.exports.Package.activateNow (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package.js:184:19)
    at C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package.js:157:32
    at Package.module.exports.Package.measure (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package.js:92:15)
    at C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package.js:150:26
    at Package.module.exports.Package.activate (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package.js:147:34)
    at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package-manager.js:548:21)
    at C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package-manager.js:529:29
    at Config.module.exports.Config.transactAsync (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package-manager.js:524:19)
    at PackageManager.module.exports.PackageManager.activate (C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\package-manager.js:506:46)
    at C:\Users\Pety\AppData\Local\atom\app-1.12.0\resources\app.asar\src\atom-environment.js:783:28

**Comment by dkniffin** [comment]
This is probably related to #24, which will be fixed by #28

**Comment by jrattray13** [comment]
should be fixed in v0.12.1 let me know if not :)

**Comment by amandeepmittal** [comment]
Yes it did. @jrattray13 Thanks for the new patch!
